src/Configuration.ts

```diff
@@ -32,10 +32,10 @@ export interface LoggingConfiguration {
 }
 
 const defaultConfiguration: Configuration = {
-  rabbitmqUrl: 'amqp://localhost',
-  mongodbUrl: 'mongodb://localhost:27017/poet',
-  ipfsUrl: 'http://localhost:5001',
-  insightUrl: 'https://test-insight.bitpay.com/api',
+  rabbitmqUrl: process.env.RABBITMQ_URL || 'amqp://localhost',
```

Comment:
The default configuration is a static immutable object that should never change.

What you want is to add these to the actually loaded configuration, which is then merged with the default configuration.

Since your introducing a new way to load configuration (env vars instead of the conf file), I'll need a bit of refactor. Let me know if you need help working on this.